app/javascript/components/applications/application_form/utils.js

```diff
@@ -0,0 +1,38 @@
+import { join, get, map, concat, pickBy } from 'lodash'
+
+export const naturalKeyFromPreference = (p) => join(
+  [
+    get(p, 'application_member.first_name'),
+    get(p, 'application_member.last_name'),
+    get(p, 'application_member.date_of_birth'),
+  ], ',')
+
+export const naturalKeyFromMember = (member) => {
+  return `${member.first_name},${member.last_name},${member.date_of_birth}`
+}
+
+export const FIELD_NAME = 'preferences'
```

Comment:
what's this?

Reply:
good catch, I think I had this file file open from a different branch and I might have hit `save all` and it was re saved in this branch. I remove this file the one under preferences is the right one